contracts/script/input/1337/eigenlayer_deployment_output.json

```diff
@@ -1,24 +1,24 @@
 {
   "addresses": {
-    "avsDirectory": "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707",
-    "avsDirectoryImplementation": "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82",
-    "baseStrategyImplementation": "0x322813Fd9A801c5507c9de605d63CEA4f2CE6c44",
-    "delayedWithdrawalRouter": "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6",
-    "delayedWithdrawalRouterImplementation": "0x959922bE3CAee4b8Cd9a407cc3ac1C251C2007B1",
-    "delegationImplementation": "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e",
-    "delegationManager": "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
-    "eigenLayerPauserReg": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
-    "eigenLayerProxyAdmin": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
-    "eigenPodBeacon": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
-    "eigenPodImplementation": "0x8A791620dd6260079BF849Dc5567aDC3F2FdC318",
-    "eigenPodManager": "0xa513E6E4b8f2a923D98304ec87F64353C4D5C853",
-    "eigenPodManagerImplementation": "0x0B306BF915C4d645ff596e518fAf3F9669b97016",
-    "emptyContract": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
-    "slasher": "0x0165878A594ca255338adfa4d48449f69242Eb8F",
-    "slasherImplementation": "0x9A676e781A523b5d0C0e43731313A708CB607508",
+    "avsDirectory": "0x0165878A594ca255338adfa4d48449f69242Eb8F",
+    "avsDirectoryImplementation": "0x9A676e781A523b5d0C0e43731313A708CB607508",
+    "baseStrategyImplementation": "0xa85233C63b9Ee964Add6F2cffe00Fd84eb32338f",
+    "delegationImplementation": "0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0",
+    "delegationManager": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
+    "eigenLayerPauserReg": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
+    "eigenLayerProxyAdmin": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+    "eigenPodBeacon": "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e",
+    "eigenPodImplementation": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
+    "eigenPodManager": "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6",
+    "eigenPodManagerImplementation": "0x959922bE3CAee4b8Cd9a407cc3ac1C251C2007B1",
+    "emptyContract": "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
+    "rewardsCoordinator": "0x8A791620dd6260079BF849Dc5567aDC3F2FdC318",
+    "rewardsCoordinatorImplementation": "0x9A9f2CCfdE556A7E9Ff0848998Aa4a0CFD8863AE",
+    "slasher": "0xa513E6E4b8f2a923D98304ec87F64353C4D5C853",
+    "slasherImplementation": "0x0B306BF915C4d645ff596e518fAf3F9669b97016",
     "strategies": "",
-    "strategyManager": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
-    "strategyManagerImplementation": "0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0"
+    "strategyManager": "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707",
+    "strategyManagerImplementation": "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82"
   },
   "chainInfo": {
     "chainId": 1337,
```

Comment:
btw did we migrate already to having different chain ids in local setup?

Reply:
I don't think so - I see either 1337 or 31337 everywhere